arrays/c-or-cpp/Array.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>

```

Comment:
Rename file's name from `Array.cpp` to `array.cpp`

Reply:
done